### DISCUSSION TITLE
raw.githubusercontent.com gives net::ERR_CONNECTION_TIMED_OUT does not load any raw files or images

### DISCUSSION THREAD

**Discussion by devarsh-mavani-19** [community/community#40603] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am trying to access raw images and files but it is not loading. I checked the google chrome console and it says  net::ERR_CONNECTION_TIMED_OUT. 
here is what i have tried so far. 

**What have i tried?**
1. Clearing cache & cookies
2. opening in incognito mode
3. changing network
4. changing browser

none of the above worked for me. It for some reason works on different device but not on my PC or my phone.

**Expected Behaviour**
It should load the resource that i am requesting

**Comment by AM-ash-OR-AM-I** [comment]
I've same exact issue, what I've found out is that this issue is there in some network providers only that too in specific area. What ISP are you using? Jio/Jio Fiber? I tried using a VPN and everything works fine, so that must mean there must be some problem with the ISP connecting to the server. This issue was there in one of my friend who also uses same ISP. However, there's another one of mine who doesn't have this issue at all with same ISP.

**Reply by devarsh-mavani-19** [reply]
I use jio sim (hotspot). I found a few old reddit posts as well. The jio users faced same issues many times in past. So i guess VPN is the solution for now.

**Reply by rishbitsnbytes** [reply]
There is one more solution to it.
Change your DNS to 1.1.1.1 and thank me later.
I was on a brink of Disconnection. Jio Backend Engineers couldn't resolve it despite multiple rounds of my network and sys logs.
But changing this DNS worked like charm!

**Reply by AM-ash-OR-AM-I** [reply]
why couldn't they do the same from their end? such simple solution works!

**Reply by rishbitsnbytes** [reply]
Well, bcoz they won't bcoz it's a public DNS owned by Cloudflare. Jio use of their own DNS which is crappy in this case. This is not the solution they will implement ever. It's just a workaround.

If you want to read more about it and then just go to 1.1.1.1
I was so fascinated when I first found this.

**Reply by AM-ash-OR-AM-I** [reply]
Yeah, indeed I visited the website and downloaded the app as well. It's just weird that Jio can't handle their own DNS, but anyways got to learn something new thanks to that, will explore more about DNS.

**Comment by rishbitsnbytes** [comment]
There is one more solution to it.
Change your DNS to 1.1.1.1 and thank me later.
I was on a brink of Disconnection. Jio Backend Engineers couldn't resolve it despite multiple rounds of my network and sys logs.
But changing this DNS worked like charm!

**Reply by shatanikmahanty** [reply]
I am pretty much new to the DNS thing. Will changing this affect browsing other website in any way?

**Reply by AM-ash-OR-AM-I** [reply]
> I am pretty much new to the DNS thing. Will changing this affect browsing other website in any way?

It won't, generally. If it does you can always revert back to the original setting. For mobile app you can download the app from https://1.1.1.1 and for desktop you can google how to change dns from terminal.

**Reply by shatanikmahanty** [reply]
Thanks for the information @AM-ash-OR-AM-I. I already changed those settings and it works like a charm. I am grateful that this issue is finally resolved for me with the help from the community ❤️

**Reply by rishbitsnbytes** [reply]
> I am pretty much new to the DNS thing. Will changing this affect browsing other website in any way?

You can read about it, but in short it is Domain Name System. Without it internet can't work. It's the first step when we surf anything on internet.
Like, as soon as we thoe google.com in browser, it first resolve the DNS by making a request to get the IP address of the server machine where google.com is hosted. Everything on internet has their own unique IP address, and servers usually have static IPs. So anyways, what it does is DNS is like phonebook in our phones, like we can't remember everyone's number, that's why we save them by their name. Human mind tends to remember alphabetical names better. And it's almost impossible to remember all the IP addresses on internet to surf google.com, Facebook.com etc. That's why it's automated by DNS. We type google.com or github.com and the first thing browser does is they go to DNS and fetch the actual IP address of these servers and then we get served from them.  Also, quick fun fact, when we make first visit to any web-app it takes a bit more time to resolve the DNS bcoz it go various processes to get actually IP. And then it stores in our system as a cache and also in out ISP's. So next visit get really fast bcoz of caching the IPs. So this was in nutshell about DNS. It's amazing how internet works, you can check more details about DNS and DNS resolver. 

So, changing your DNS to 1.1.1.1 only makes this difference that all the IPs will be fetched from Cloudflare's server instead if JIO's. As AM said, generally it won't make any difference. Cloudflare's is almost always better. But you can always go back.

**Reply by aasisodiya** [reply]
Thanks, @rishbitsnbytes for the solution. I recently started facing this issue
